Makefile.am

```diff
@@ -86,6 +81,9 @@ persistentcachedir=$(localstatedir)/cache/metrics/
 configdir=$(sysconfdir)/metrics/
 permissions_file=$(configdir)eos-metrics-permissions.conf
 
+# systemd
+systemdsystemunit_DATA = 
```

Comment:
What's this doing here?

Reply:
We append to it later in one of the subdirectories included by this makefile